frontend/src/pages/HomePage/components/InfinityCarousel/index.tsx

```diff
@@ -11,40 +15,43 @@ interface InfinityCarouselProps {
   slideList: Slide[];
 }
 
-const REAL_START_INDEX = 1;
+interface HandleSlideAnimationParams {
+  slide: HTMLDivElement;
+  withTransition: boolean;
+  index: number;
+  deltaX?: number;
+}
 
-const TRANSITION_DURATION = 500; // NOTE: 트랜지션(애니메이션) 시간
-const AUTO_SLIDE_INTERVAL = 6000; // NOTE: 자동 슬라이드 시간
+const REAL_START_INDEX = 1;
+const TRANSITION_DURATION = 500; // 트랜지션 시간
+const AUTO_SLIDE_INTERVAL = 6000; // 자동 슬라이드 시간
+const DRAG_THRESHOLD = 0.3; // 슬라이드 넘기기 위한 최소 드래그 거리 비율 (슬라이드 너비의 30%)
 
 const InfinityCarousel = ({ slideList }: InfinityCarouselProps) => {
-  // NOTE: 마지막 슬라이드를 복제해서 slideList의 맨 앞에 추가하므로 처음에 보여져야 하는 슬라이드는 1번 인덱스
   const [currentSlideIndex, setCurrentSlideIndex] = useState(REAL_START_INDEX);
   const [isTransitioning, setIsTransitioning] = useState(false);
   const [clicked, setClicked] = useState(false);
+  const [isDragging, setIsDragging] = useState(false);
+  const [startX, setStartX] = useState(0); // 드래그를 시작한 시점의 X 좌표
+  const [deltaX, setDeltaX] = useState(0); // 현재 드래그 중인 위치와 시작 위치 사이의 차이
+
   const slideRef = useRef<HTMLDivElement>(null);
 
   const slideLength = slideList.length;
-  // NOTE: 첫 슬라이드와 마지막 슬라이드의 복제본을 각각 맨 뒤, 맨 처음에 추가
   const clonedSlideList = [slideList[slideLength - 1], ...slideList, slideList[0]];
 
   const scrollToSlide = (index: number, withTransition = true) => {
     if (slideRef.current) {
       setIsTransitioning(true);
-      window.requestAnimationFrame(() => handleSlideAnimation({ slide: slideRef.current!, withTransition, index }));
+      handleSlideAnimation({ slide: slideRef.current, withTransition, index });
     }
     setCurrentSlideIndex(index);
   };
 
-  interface HandleSlideAnimationParams {
-    slide: HTMLDivElement;
-    withTransition: boolean;
-    index: number;
-  }
-
-  const handleSlideAnimation = ({ slide, withTransition, index }: HandleSlideAnimationParams) => {
+  const handleSlideAnimation = ({ slide, withTransition, index, deltaX = 0 }: HandleSlideAnimationParams) => {
     const slideWidth = slide.clientWidth;
     slide.style.transition = withTransition ? `transform ${TRANSITION_DURATION}ms ease-in-out` : 'none';
-    slide.style.transform = `translate3d(-${slideWidth * index * 0.1}rem, 0, 0)`;
+    slide.style.transform = `translate3d(${-slideWidth * index + deltaX}px, 0, 0)`;
```

Comment:
`deltaX / 10` 으로 변경하고 px을 rem으로 변경하니 슬라이드 이미지가 보이지 않는 이슈가 있어서 일단 px로 했습니다

Reply:
```suggestion
    slide.style.transform =`translate3d(${(-slideWidth * index + deltaX) * 0.1}rem, 0, 0)`
```
이렇게 하면 rem 사용할 수 있어요. 확인 부탁드려요

Reply:
반영 완료! 잘 돌아갑니다!